### PR TITLE
[dev] Update leeway

### PR DESCRIPTION
**Pull request by aledbf**
**Changes:**
- Bump Go version to 1.16.5
- Bump nvm.sh to v0.38.0
- Bump leeway to v0.2.2 (support for test coverage file generation)

fixes https://github.com/gitpod-io/gitpod/issues/4379

requires PR https://github.com/gitpod-io/workspace-images/pull/411

### CODE REVIEW

File: dev/image/Dockerfile [hunk 1]
```diff
@@ -2,9 +2,9 @@
 # Licensed under the GNU Affero General Public License (AGPL).
 # See License-AGPL.txt in the project root for license information.
 
-FROM gitpod/workspace-full-vnc:latest
+FROM gitpod/workspace-full-vnc:branch-aledbf-precommit
```
Comment (aledbf) [inline dev/image/Dockerfile:5]: Until https://github.com/gitpod-io/workspace-images/pull/411 is merged

**Review by JanKoehnlein — APPROVED** [review]
LGTM

File: WORKSPACE.yaml [hunk 2]
```diff
@@ -9,7 +9,7 @@ defaultArgs:
 defaultVariant:
   config:
     go:
-      lintCommand: ["golangci-lint", "run", "--disable", "govet,errcheck,typecheck,staticcheck", "--allow-parallel-runners"]
+      lintCommand: ["golangci-lint", "run", "--disable", "govet,errcheck,typecheck,staticcheck", "--allow-parallel-runners", "--timeout", "5m"]
```
Comment (aledbf) [inline WORKSPACE.yaml:12]: (default timeout is 1m)

**Review by csweichel — APPROVED** [review]
Thanks for including the qemu changes :)

LGTM.

**Pull request merged**